### PR TITLE
update installcourse

### PR DESCRIPTION
**Pull request by justinnoah**

### CODE REVIEW

File: local/orvsd_installcourse/lib.php [hunk 1]
```diff
@@ -101,7 +101,7 @@ function orvsd_installcourse_update($event_data) {
     $external_token->userid = 2;
     $external_token->contextid = 1;
     $external_token->creatorid = 2;
-    $external_token->iprestriction = "140.211.167.136/31,140.211.15.0/24";
+    $external_token->iprestriction = "140.211.167.136/31,140.211.15.0/24,10.0.0.0/8";
```
Comment (armiller) [inline local/orvsd_installcourse/lib.php:104]: Does this really need to be open to the entire backend network?

Comment (justinnoah) [inline local/orvsd_installcourse/lib.php:104]: We need to allow the OpenStack VMs and the workstation Vagrant boxes to access at least the test27/test25/test22 sites.

Comment (ramereth) [inline local/orvsd_installcourse/lib.php:104]: We should be able to size that down to 10.1.0.0 /23 we normally use. Do you know what IP the vagrant boxes query from? I think its just one IP (probably 10.2.x.x something).

Comment (justinnoah) [inline local/orvsd_installcourse/lib.php:104]: It looks like 10.0.2.15

**Pull request closed without merging**